packages/solid/components/Badge/Badge.tsx

```diff
@@ -42,10 +42,15 @@ const Badge: Component<BadgeProps> = (props: BadgeProps) => {
   const tone = createMemo(() => props.tone ?? styles.tone);
 
   return (
-    <BadgeContainer {...props} height={props.height || styles.Text.base.lineHeight} tone={tone()} style={props.style}>
+    <BadgeContainer
+      {...props}
+      height={props.height || styles.Text.base.lineHeight}
+      tone={tone()}
+      style={props.style}
+    >
       <Text
         color={props.textColor}
-        lineHeight={props.height || styles.Text.base.lineHeight}
+        lineHeight={(props.height || styles.Text.base.lineHeight) + 2}
```

Comment:
is this +2 a fix for a specific font? if so we might want to derive this from a style instead of a hardcoded offset

Reply:
It was added because of the border around it adds an extra 2 px - so really that 2 is borderWidth..